src/qt/walletview.cpp

```diff
@@ -273,7 +273,7 @@ void WalletView::backupWallet()
 {
     QString filename = GUIUtil::getSaveFileName(this,
         tr("Backup Wallet"), QString(),
-        tr("Wallet Data (*.dat)"), nullptr);
+        tr("Wallet Data") + QLatin1String(" (*.dat)"), nullptr);
```

Comment:
Also needs annotation...

(If you were asked to translate "Wallet Data" with no context, would you know what it was referring to? XD)

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/220#issuecomment-792023727).